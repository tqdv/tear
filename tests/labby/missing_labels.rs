use tear::twist;

fn missing_labels() {
	twist! { -labby JUST_BREAK }
}

fn main () {}