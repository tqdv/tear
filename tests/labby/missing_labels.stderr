error: Missing `|` separator after labels in `twist! -labby` macro invocation. Add labels, or use `twist!` without `-labby`.
 --> $DIR/missing_labels.rs:4:2
  |
4 |     twist! { -labby JUST_BREAK }
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
